src/components/skip-link/_skip-link.scss

```diff
@@ -5,7 +5,8 @@
 @include govuk-exports("govuk/component/skip-link") {
   .govuk-skip-link {
     @include govuk-visually-hidden-focusable;
-    @include govuk-link-common;
+    @include govuk-typography-common;
+    @include govuk-focusable-fill;
```

Comment:
Why is this using govuk-focusable-fill and why is the skip link styled differently to other links?

Reply:
This is a design decision from @dashouse to not use the heavy underline focus state for skip-link.

Reply:
🤔 ok, currently this component and Footer are the only two components still using `govuk-focusable-fill` mixin. When we come to review the public mixing perhaps we can just hard code the mixins output here instead of having it hanging around

Reply:
Yeah once we've got everything production ready we can (after the audit potentially) decide on how the mixins should work, with the new guidance in mind...